Source/WebCore/style/StyleAdjuster.cpp

```diff
@@ -904,6 +904,26 @@ void Adjuster::adjustForSiteSpecificQuirks(RenderStyle& style) const
         }
     }
 #endif
+#if PLATFORM(IOS_FAMILY)
+    if (m_element->document().quirks().shouldInjectYouTubeFullsceenStyles()) {
```

Comment:
this might be a bit better if the if condition was inverted, and then just do an early return (to avoid nesting too much)

Reply:
To be fair, that's not in line with the other quirks here, and would make it more difficult to add a new quirk below this one.